src/components/home/TimeTracker.tsx

```diff
@@ -163,7 +50,6 @@ export default TimeTracker;
 
 const CustomContainer = styled.div`
   display: flex;
-  flexDirection: row;
```

Comment:
is this CSS setting already taken care of by an inherited class or did you just find it's unnecessary to get the same behavior?

Reply:
flexDirection is an invalid property, so it wasn't actually doing anything.  flex-direction is correct.  But flex-direction: row is default, so it shouldn't be needed.